.ci/Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM ubuntu:16.04
+
+ENV HADOOP_HOME /usr/local/hadoop
+ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64
+
+# install packages
+RUN \
+  apt-get update && apt-get install -y \
+  ssh \
+  rsync \
+  vim \
+  openjdk-8-jdk
+
+# download and extract hadoop, set JAVA_HOME in hadoop-env.sh, update path
+RUN \
+  wget http://apache.mirrors.tds.net/hadoop/common/hadoop-3.1.2/hadoop-3.1.2.tar.gz && \
+  tar -xzf hadoop-3.1.2.tar.gz && \
+  rm -rf hadoop-3.1.2.tar.gz && \
+  mv hadoop-3.1.2 /usr/local && \
+  ln -sf /usr/local/hadoop-3.1.2/ /usr/local/hadoop
+
+CMD cat hadoop_config >> ~/.bashrc
```

Comment:
You need to `COPY` hadoop_config into the container first

Reply:
I've included the following line:
```
ADD configs/hadoop_config /usr/local/hadoop
CMD cat /usr/local/hadoop/hadoop_config >> ~/.bashrc
```
Is this fine?